app/Http/Controllers/Maps/MapDataController.php

```diff
@@ -487,9 +487,12 @@ public function getDevices(Request $request): JsonResponse
                 $updowntime = '';
             }
 
+            $deviceTypes = collect(LibrenmsConfig::get('device_types', []));
+
             $device_list[$device->device_id] = [
                 'id' => $device->device_id,
                 'icon' => $device->icon,
+                'typeIcon' => $deviceTypes->firstWhere('type', $device->type)['icon'] ?? null,
```

Comment:
Is it not better to do `?? 'server'` here and not do it in the blade files later?